.github/workflows/inductor-perf-test-nightly.yml

```diff
@@ -58,6 +58,11 @@ on:
         required: false
         type: string
         default: inductor_huggingface_perf,inductor_timm_perf,inductor_torchbench_perf
+      caching:
```

Comment:
You would either need to add `cachebench` in the list of `inductor_huggingface_perf,inductor_timm_perf,inductor_torchbench_perf`, or just run it always.  The problem here is that you won't be able to add another parameter here as the list of parameter has reached the max size that GitHub allows (10 parameters).  I know it's weird, but it's what it is.